create_jenkins_job.py

```diff
@@ -160,7 +160,7 @@ def main(argv=None):
     os_config_overrides = {
         'linux-rhel': {
             'mixed_overlay_pkgs': '',
-            'ignore_rmw_default': {'rmw_connext_cpp', 'rmw_connext_dynamic_cpp', 'rmw_opensplice_cpp'},
+            'ignore_rmw_default': {'rmw_connext_cpp', 'rmw_connext_dynamic_cpp', 'rmw_connextdds'},
```

Comment:
This is technically removing opensplice, but since we are never going to target Dashing with RHEL, this is fine.

Reply:
Oops, missed this one, sorry.

Reply:
I don't have a strong preference and I'm not even sure it would matter (since I'm not a maintainer :) ), but your second suggestion looks like a good way to be consistent with other parts of the script, and to be more flexible going forward by automatically detecting `ignore_rmw_default` from the command line.

I can apply whatever fix you select, although it might be easier to do a "maintainer change" (is that the term?) like @clalancette did earlier since you have push access to the branch on this repo, than me opening another PR to incorporate changes.

Reply:
I'll go make the change to change this back to the old code; that is maximally compatible (if not optimal).